src/main/java/core/basesyntax/db/FileWriterImpl.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax.db;
+
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class FileWriterImpl {
+    public void write(String report, String fileName) {

```

Comment:
The method signature lacks access modifier. It's good practice to explicitly declare the access level of your methods

Reply:
why "public" modifier is not correct here ?